bot/client.js

```diff
@@ -15,6 +15,22 @@ client.on('ready', () => {
   util.log('Bot Online and Ready', 0);
 });
 
+// Actions for new guild members
+client.on('guildMemberAdd', (member) => {
+  // Send a the direct welcome message to a guild member
+  member.send(`Welcome ${member} to the Full Sail Armada! Please read our Terms of Service below to get started: \n\n --Terms of Service--`);
+  util.log('Direct Message Sent to New Member', 0);
+
+  // Variable to store channel if it exists
+  const channel = client.channels.find('name', 'general');
+  // Check if channel exists
+  if (channel) {
+    // Send a welcome broadcast tot he general channel
+    channel.send(`Welcome to the Full Sail Armada ${member}!`);
```

Comment:
Proper sentence structure would be idea.

Welcome to the Full Sail Armada```,``` ${member}!